pysces/panel.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import division
 
 import numpy as np
+import sys
+import matplotlib.pyplot as plt
```

Comment:
Looks like this is why the Travis build is failing: matplotlib is now imported.  I'd rather not make matplotlib a dependency, and it doesn't appear it is needed here, so probably best to remove this.